server/src/utilities/adminInit.ts

```diff
@@ -0,0 +1,12 @@
+const admin = require('firebase-admin');
+const serviceAccount = require("../../src/private_key/private.json");
+
+export const adminApp = admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+});
+
+export const db = admin.firestore();
+export const connectedUsersCollection = db.collection('users');
```

Comment:
why are we reverting this back to `users` instead of the `ConnectedUsers` collection added to the docs?

Reply:
Will change :+1: